5-predict_subtype_reconstructed_data.R

```diff
@@ -74,25 +74,33 @@ for (seed in filename.seeds) {
       
       # read in reconstructed data from current platform and reconstruction 
       # method
-      file.identifier <- paste(plt, rcn, seed, sep = "_")
+      file.identifier <- paste(rcn, plt, seed, sep = "_")
       recon.rds <- recon.files[grep(file.identifier, recon.files)]
       recon.list <- readRDS(recon.rds)
       
       # get confusionMatrix objects
-      plt.list[[rcn]] <- PredictReconDataWrapper(train.list = train.list,
-                                                 recon.list = recon.list,
-                                                 sample.df = sample.df)
+      plt.list[[rcn]] <- PredictWrapper(train.model.list = train.list,
```

Comment:
would it be possible to only run this once and get Kappa every time? Meaning, Kappa is always calculated but it is added in a separate attribute of the return variable. This way you don't have to run PredictWrapper twice in this loop. Not major benefits if PredictWrapper is quick